lib/locales/en.ts

```diff
@@ -944,13 +965,17 @@ export const locale: Locale = {
     thorlac_of_iceland_bishop: 'Saint Thorlac, Bishop',
     thursday_of_the_lords_supper: 'Thursday of the Lord’s Supper',
     timothy_of_ephesus_and_titus_of_crete_bishops: 'Saints Timothy and Titus, Bishops',
+    timothy_of_ephesus_and_titus_the_first_bishop_of_crete_bishops:
+      'Saint Titus, the first Bishop of Crete, Saint Timothy, Bishops',
     transfiguration_of_the_lord: 'The Transfiguration of the Lord',
     translation_of_the_relics_of_odile_of_alsace_abbess: 'Translation of the Relics of Saint Odile',
     translation_of_the_relics_of_saint_stephen_of_hungary: 'Translation of the Relics of Saint Stephen of Hungary',
     turibius_of_mogrovejo_bishop: 'Saint Turibius of Mogrovejo, Bishop',
+    twelve_apostles: 'The Holy Twelve Apostles',
```

Comment:
That one does not have a matching calendar definition. Is the calendar definition missing? Or is it a locale that should be applied to an existing element but only for `greece.ts`?